packages/jellyfin_media_player.rb

```diff
@@ -56,11 +58,11 @@ def self.build
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} #{CREW_NINJA} -C builddir install"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d"
-    File.write "#{CREW_DEST_PREFIX}/etc/env.d/10-jellyfinmediaplayer", <<~JELLYFIN_ENVD_EOF
+    File.write 'jellyfinmediaplayer_env.d', <<~JELLYFIN_ENVD_EOF
       # QT_QPA_PLATFORM=wayland is unaccelerated and unusable.
       alias jellyfinmediaplayer="QT_QPA_PLATFORM=eglfs jellyfinmediaplayer"
```

Comment:
Would a shorter alias make it easier?  Something like jmp?

Reply:
Yes! I can change that...

Reply:
Fixed...